frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -656,6 +691,69 @@ def map_wires(self, wire_map):
         return self
 
 
+class Controlled:
+    """Similar to the Adjoint class, it provides near identical behaviour as PennyLane for
+    ctrl instances with only a single base operation. Additionally, it provides the same
+    functionality as HybridCtrl."""
+
+    def __new__(cls, base, *_, **__):
+        if isinstance(base, Operation):
+            return object.__new__(ControlledOp)
+
+        return object.__new__(ControlledBase)
+
+    # pylint: disable=too-many-arguments,abstract-method
+    def __init__(
+        self, base, tracing_artifacts=None, control_wires=None, control_values=None, work_wires=None
+    ):
+        super().__init__(
+            base,
+            control_wires,
+            control_values=control_values,
+            work_wires=work_wires,
+        )
+
+        # Added this condition to support direct calls of this class outside the QJIT context
+        # in PL integrated tests:
+        if tracing_artifacts:
+            HybridCtrl.__init__(
+                self,
+                *tracing_artifacts,
+                control_wires=control_wires,
+                control_values=control_values,
+                work_wires=work_wires,
+            )
```

Comment:
Actually I had to use this if-condition to get `Controlled._flatten` PL tests pass when constructing a `Controlled` class outside the context of QJIT; the problem here is that there is not default value for `in_classical_tracers`, `out_classical_tracers`, and `regions`. But you have a good point here and the alternative approach that would fix the issue is 
 
```
            HybridCtrl.__init__(
                self,
                *tracing_artifacts if tracing_artifacts else ([], [], []),
                control_wires=control_wires,
                control_values=control_values,
                work_wires=work_wires,
            )

```
what do you think?

Reply:
Default values can be produced in `trace_body` no?

Reply:
> tracing_artifacts if tracing_artifacts else ([], [], [])

This doesn't seem quite right either because that means the HybridOp has no nested tape? Even if a HybridOp is generated outside a qjit context, it may not have input or output tracers but it should still have a nested region 🤔

Reply:
Wait I think your existing implementation already produces  `[], [], [region]` when used outside of a qjit context. Are you sure `tracing_artifacts` can ever be False?

Reply:
Ah, there was an issue with the `_unflatten` method. Fixed it! [cf70d1b](https://github.com/PennyLaneAI/catalyst/pull/771/commits/cf70d1b140e7417ff158dc5a43e930b053c17258)